config/samples/olm_v1alpha1_extension.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: olm.operatorframework.io/v1alpha1
+kind: Extension
+metadata:
+  name: extension-sample
+  namespace: extension-namespace
+spec:
+  paused: false
+  serviceAccountName: extension-sa
+  defaultNamespace: extension-sample
+  package:
```

Comment:
Was working based of this PR, realized that this sample need to be updated according to the recent changes made in the spec.

Reply:
Basically this:
```yaml
apiVersion: olm.operatorframework.io/v1alpha1
kind: Extension
metadata:
  labels:
    app.kubernetes.io/name: extension
    app.kubernetes.io/instance: extension-sample
    app.kubernetes.io/part-of: operator-controller
    app.kubernetes.io/managed-by: kustomize
    app.kubernetes.io/created-by: operator-controller
  name: extension-sample
  namespace: extension-sample
spec:
  paused: false
  serviceAccountName: extension-sa
  source:
    package:
      name: argcocd-operator
      version: <>
```

Reply:
Not the labels, just the spec. Removal of default namespace, embedding package inside source and also renaming paused to managed.
I was trying to use this sample to test extensions with App CR, which is when realized that some of the fields are not upto date with the what is present in CRD.